service/ziti-tunnel/service/ipc.go

```diff
@@ -1196,27 +1167,36 @@ func handleEvents(isInitialized chan struct{}) {
 				notificationMessage := ""
 
 				var notificationMinTimeout int32 = 0
-				var notificationMaxTimeout int32 = 0
-				if (id.CId.MaxTimeout > -1) && (id.CId.MaxTimeout-int32(time.Since(id.CId.LastUpdatedTime).Seconds())) < 0 {
+				var notificationMaxTimeout int32 = -1
+				if (id.CId.MfaMaxTimeout > -1) && (id.CId.MfaMaxTimeout-int32(time.Since(id.CId.MfaLastUpdatedTime).Seconds())) < 0 {
```

Comment:
suggestion: this code would read much cleaner if we added a function to `cziti.ZIdentity` which determined this state. Here it's kind of buried in this case statement.